endpoints/put.go

```diff
@@ -184,6 +186,12 @@ func logBackendError(err error) {
 func (e *PutHandler) handle(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	e.metrics.RecordPutTotal()
 
+	// If incoming request comes with a referer header, there's a e.cfg.refererLogRate percent chance
+	// getting it logged
+	if referer := r.Referer(); referer != "" && utils.RandomPick(e.cfg.refererLogRate) {
+		logrus.Info("PUT request Referer header: " + referer)
```

Comment:
Is this line being tested in `put_test.go`? I don't see coverage. Is it worth adding a test similar to my recommendation in `get_test.go`?